site/base.less

```diff
@@ -6,7 +6,7 @@ html, body {
   -ms-text-size-adjust: 100%;
   margin: 0;
   padding: 0;
-  -webkit-font-smoothing: antialiased;
+  -webkit-font-smoothing: subpixel-antialiased;
```

Comment:
BTW, I was looking at this some more and I kind of think this should just be removed all together: http://usabilitypost.com/2012/11/05/stop-fixing-font-smoothing/